<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaxiViewClass</class>
 <widget class="QFrame" name="MaxiViewClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>82</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MaxiView</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>14</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="actBtnLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="act1Button">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Act1</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">activitiesButtonsGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="act2Button">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Act2</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">activitiesButtonsGroup</string>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="idleButton">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>60</width>
         <height>60</height>
        </size>
       </property>
       <property name="text">
        <string>Idle</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">activitiesButtonsGroup</string>
       </attribute>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="activitiesButtonsGroup"/>
 </buttongroups>
</ui>
